python/deploy_tools.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
не у всех bash живет в /bin

Reply:
справедливо!

готово